.github/workflows/NodeCI.yml

```
@@ -44,9 +44,19 @@ jobs:
                   node-version: 12.13.x
             - name: Install Target Packages
               run: |+
+                  npm install
```

One small nit. Let's use `npm ci`, it's faster. (Same with v8.x)

```suggestion
                  npm ci
```

---

Thank you for pointing out 😅